<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>824</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>DisplayKeys-IS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ProjectFiles/assets/DisplayKeys-IS.ico</normaloff>ProjectFiles/assets/DisplayKeys-IS.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#left_menu_wrapper {
	background-color: #22282c;
}

#preview_wrapper {
	background-color: #292f33;
}

* {
	font: &quot;Poppins&quot;;
}

QLabel {
color: white;
}
QPushButton{
	border-radius: 10px;
	background-color: rgb(65, 75, 81);
	color: white;
}
QPushButton:hover {
    background-color: rgb(82, 95, 102);
    border: 2px solid #AAAAAA;
}
QComboBox {
	border-radius: 10px;
	background-color: rgb(65, 75, 81);
	color: white;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="left_menu_wrapper" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>30</number>
       </property>
       <item>
        <widget class="QLabel" name="title_label">
         <property name="font">
          <font>
           <family>Poppins</family>
           <pointsize>18</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Image Splitter made by Neuffexx</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="import_image_wrapper" native="true">
         <property name="styleSheet">
          <string notr="true">.QWidget {
border-bottom: 2px solid grey
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="browse_label">
            <property name="font">
             <font>
              <family>Poppins</family>
              <pointsize>18</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Choose Image</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="input_image_path">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browse_button">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Browse image</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="output_wrapper" native="true">
         <property name="styleSheet">
          <string notr="true">.QWidget {
border-bottom: 2px solid grey
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="browse_label_2">
            <property name="font">
             <font>
              <family>Poppins</family>
              <pointsize>18</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Choose Output Location</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="input_image_path_2">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browse_button_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Browse folder</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="splitting_wrapper" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QLabel" name="splitting_title">
            <property name="font">
             <font>
              <family>Poppins</family>
              <pointsize>18</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Set Splitting Paramters</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="minimumSize">
             <size>
              <width>162</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Poppins</family>
              <pointsize>15</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox {
    background-color: #FFFFFF;
    border: 1px solid #CCCCCC;
    border-radius: 10px;
    padding: 5px;
    min-width: 150px;
    color: #333333;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: #CCCCCC;
    border-left-style: solid;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #EEEEEE, stop:1 #FFFFFF);
}

QComboBox::down-arrow {
    image: url(assets/chevron-down.svg)
}

QComboBox:hover {
    background-color: #F2F2F2;
    border: 1px solid #AAAAAA;
}

QComboBox::drop-down:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #DDDDDD, stop:1 #FFFFFF);
}</string>
            </property>
            <property name="currentText">
             <string>Default</string>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>Default</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>User Defined</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="settings_wrapper" native="true">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QSpinBox {
    border: 1px solid #d1d1d1;
    border-radius: 10px;
    padding: 5px;
    background-color: #f5f5f5;
    color: #333333;
}

QSpinBox:hover {
    background-color: #e8e8e8;
}

QSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right;
    width: 16px;
    border-top-right-radius: 10px;
}

QSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right;
    width: 16px;
    border-bottom-right-radius: 10px;
}

QSpinBox::up-arrow {
    image: url(assets/chevron-up.svg);
    width: 10px;
    height: 10px;
}

QSpinBox::down-arrow {
    image: url(assets/chevron-up.svg);
    width: 10px;
    height: 10px;
}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="rows_label">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Rows</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="rows_box">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="columns_label">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Columns</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="columns_box">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="gap_label">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Gap ( in pixels )</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="gap_box">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="split_image_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Split Image</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="preview_wrapper" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="topMargin">
        <number>150</number>
       </property>
       <property name="bottomMargin">
        <number>150</number>
       </property>
       <item>
        <widget class="QWidget" name="preview_widget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
